support/cas-server-support-shibboleth/src/main/java/org/apereo/cas/support/saml/ShibbolethIdPEntityIdAuthenticationServiceSelectionStrategy.java

```diff
@@ -32,6 +36,9 @@
     private final int order = Ordered.HIGHEST_PRECEDENCE;
     private final transient ServiceFactory webApplicationServiceFactory;
     private final String idpServerPrefix;
+    private final ServicesManager servicesManager;
+
+    private static final String PARAMETER_ENTITY_ID = "entityId";
```

Comment:
It might actually be good to break up the saml-core module into one that has the config/starter beans and one that just has the API components so that this and anywhere else could take advantage of the API without activating beans. (Not required here; future)